codecov.yaml

```diff
@@ -11,13 +11,25 @@ comment:
   after_n_builds: 1
   show_carryforward_flags: true
 
+github_checks:
+  annotations: true
+
 coverage:
   status:
+    patch:
+      default:
+        target: 60%
+        threshold: 0.1%
```

Comment:
what does `threshold` mean here if we provide an actual `target` number (and not `auto`)?

Reply:
I think setting both `target` and `threshold` in patch status means that the patch unit-test coverage must reach [(`target` - `threshold`) ~ `target`].

For example, if I set the `target` as 34.33% and the `threshold` as 1%, the patch unit-test coverage result between 33.33% and 34.33% will be accepted.
![image](https://user-images.githubusercontent.com/25790455/180404432-4020f815-becf-4cc4-8dd3-6f814046e596.png)

However, just setting a `target` here is also reasonable.

Reply:
yes, I am not sure I see the point, because we could just set `target` to `33.33%` in the first place?

Reply:
In my opinion,  this is some kind of buffer mechanism. Some code is easy to add unit-test, its coverage can reach 100% for example, also some code is hard to add unit-test, but it must achieve the lowest coverage of 60%. Thus the total unit test coverage of the project can reach 80%.
So, setting the `target` to 100%, and setting the `threshold`  to 40% is more reasonable than just setting `target` to 80%.
In addition, the result display of the code-cov check is different between `target` and `within a threshold of the target`.
It is clear and can serve as reviewing references, I think.
![image](https://user-images.githubusercontent.com/25790455/180690938-1185949d-fec1-4d5a-b9ae-85f87426c831.png)